src/runtime-rs/fuzz/README.md

```diff
@@ -0,0 +1,25 @@
+# Fuzz testing
+To get started with fuzz testing you'll need to install cargo-fuzz.
+
+`cargo install cargo-fuzz`
+
+You can list the available fuzz harnesses using cargo fuzz. e.g.
+```
+$ cd src/runtime-rs/fuzz
+$ cargo +nightly fuzz list
```

Comment:
I'm wondering if any Rust stable versions can be used instead of nightly versions to run cargo fuzz list ？
I am concerned that we should minimize the use of nightly versions as much as possible.

Reply:
+1 For using rust stable versions.

Reply:
It turns out you can use stable for `cargo fuzz list`, but nightly is [still required for building the fuzzer's](https://rust-fuzz.github.io/book/cargo-fuzz/setup.html?highlight=nightly#requirements). The reason nightly should be used is because it allows for certain instrumentation flags that are not available on stable. It's worth pointing out however that the fuzzer's don't use nightly API's and as a result *can* be built on stable. It's just that without the instrumentation the fuzzer ends up being next to useless, as the fuzzing engine will use the [coverage instrumentation to guide it's inputs](https://llvm.org/docs/LibFuzzer.html#id12), as well as various other things.